doc/tests.rst

```diff
@@ -674,6 +674,8 @@ The ``cram`` stanza accepts the following fields:
 - ``deps`` - dependencies of the test
 - ``(package <package-name>)`` - attach the tests selected by this stanza to the
   specified package
+- ``(runtest_alias <true|false>)`` - when set to ``false``, do not add the
+  tests to the ``runtest`` alias by default.
```

Comment:
I don't get why "by default" is there. Can you add a note that if this field is omitted then the tests will be included in the `@runtest` alias. Also what's the use case for `(runtest_alias true)`? Is there ever a time when the field should be explicitly set to true?

Reply:
> I don't get why "by default" is there. Can you add a note that if this field is omitted then the tests will be included in the @runtest alias

I clarified it.

> Also what's the use case for (runtest_alias true)? Is there ever a time when the field should be explicitly set to true?

Mostly to allow things like `(runtest_alias %{lib-available:foo})` or blang expressions if there is ever a need. 

Experience tells me that flag-like fields (e.g. `no_runtest_alias`) end up being annoying to use. E.g. suppose that we change the default to not add the runtest alias to cram by default in a later version of the dune language, how would the user be able to switch it back on?

Reply:
Makes sense. Thanks for clarifying.